src/Symfony/Component/Asset/Context/NullContext.php

```diff
@@ -18,6 +18,13 @@
  */
 class NullContext implements ContextInterface
 {
+    public function __construct()
+    {
+        if (__CLASS__ !== get_class($this)) {
```

Comment:
@theofidry Symfony does not use ``@public``

@GuilhemN As this class does not have a constructor in the current codebase, people extending it currently **cannot** be calling the parent constructor (this would be a fatal error as there are none)